me.stonegate.tsacdop.json

```
@@ -0,0 +1,65 @@
+{
+    "app-id": "me.stonegate.tsacdop",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "40",
+    "sdk": "org.gnome.Sdk",
+    "command": "tsacdop",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
```

```suggestion
        "--socket=fallback-x11",
        "--socket=wayland",
```

assuming this works with wayland.